espnet2/bin/enh_scoring.py

```diff
@@ -63,18 +63,14 @@ def scoring(
             if ref.ndim > inf.ndim:
                 # multi-channel reference and single-channel output
                 ref = ref[..., ref_channel]
-                assert ref.shape == inf.shape, (ref.shape, inf.shape)
             elif ref.ndim < inf.ndim:
                 # single-channel reference and multi-channel output
-                raise ValueError(
-                    "Reference must be multi-channel when the \
-                    network output is multi-channel."
-                )
+                inf = inf[..., ref_channel]
```

Comment:
@Emrys365 I changed this because on evaluation only anechoic signals from channel 0 are provided in this dataset.

Reply:
Oh I see. Indeed, this case should be supported in evaluation.